src/thunderbots/software/backend_output/grsim/grsim_backend.h

```diff
@@ -46,10 +46,17 @@ class GrSimBackend : public Backend
      *
      * @param angular_velocity The angular velocity to set for the robot, in Radians per
      * second.
+     * @param kick_power How hard to kick the ball, in meters/second. If chip is false,
+     * the ball is kicked along the ground. If chip is true, the ball is chipped in
+     * the air
```

Comment:
I used `_power` because it was more generic, since it could technically refer to a kick (on the ground) or a chip. Because grSim just treats both with `m/s` I think what you suggested is fine.

I think `AngularVelocity` is correct here because the robot can rotate in either direction and therefore seems more appropriate than `AngularSpeed`. A negative values causes counter-clockwise rotation, and vice-versa for positive values.

Reply:
:+1:  to everything above. I was thinking of `LinearSpeed` as just a typedef of a double (for now) to make this a bit more clear.

Reply:
Would prefer not to use a one-of typedef, so will leave it as a double with a better variable name and better comments.